Docker image fails to build

Building the docker image fails with: `Launcher Chrome not found. Not installed?`

Details from running `docker build .`:
```
Step 8/13 : RUN npm test
 ---> Running in 80e4d7e4777b

> datadelivery-ui@1.3.1 test /src
> ember test

- Building
Environment: test
- Building
- cleaning up
- cleaning up
cleaning up...
Built project successfully. Stored in "/src/tmp/class-tests_dist-xdHl6a9K.tmp".
not ok 1 [undefined ms] - Error
    ---
        message: >
            Launcher Chrome not found. Not installed?
    ...

1..1
# tests 1
# pass  0
# skip  0
# fail  1
Launcher Chrome not found. Not installed?


Stack Trace and Error Report: /tmp/error.dump.ee728d7757df3ff34c7c3aab27402a9f.log
npm ERR! Test failed.  See above for more details.
The command '/bin/sh -c npm test' returned a non-zero code: 1
```

The`circleci/node:8.9-browsers` image sets USER to be `circleci`:
https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/node/images/8.9.4/Dockerfile#L81

This causes permission issues when running `npm install`
```
npm WARN checkPermissions Missing write access to /src
npm ERR! path /src
npm ERR! code EACCES
```
Switch user back to root seems to fix this.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -15,7 +15,6 @@ jobs:
           command: |
             npm config set spin false
             npm install
-            npm install phantomjs-prebuilt
 
       - save_cache:
           paths:

--- a/Dockerfile
+++ b/Dockerfile
@@ -7,7 +7,6 @@ ADD package.json /src/package.json
 ADD package-lock.json /src/package-lock.json
 RUN npm install
 ADD . /src/
-
 # add CI flag so chrome --no-sandbox flag will be used
 RUN CI=true npm test
 RUN npm run build -- --environment production

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,9 +3,18 @@ WORKDIR /src
 ADD package.json /src/package.json
 ADD package-lock.json /src/package-lock.json
 RUN npm install
-RUN npm install phantomjs-prebuilt
+# Install latest chrome package so we can run tests
+# Copied from https://github.com/GoogleChromeLabs/lighthousebot/blob/master/builder/Dockerfile
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-unstable --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /src/*.deb
 ADD . /src/
-RUN npm test
+
+# add CI flag so chrome --no-sandbox flag will be used
+RUN CI=true npm test
 RUN npm run build -- --environment production
 
 FROM nginx:1.13

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,16 +1,11 @@
-FROM node:8 as builder
+FROM circleci/node:8.9-browsers as builder
+# The base image sets USER to circleci.
+# Reset user back to root so we have permissions for the ADDed files.
+USER root
 WORKDIR /src
 ADD package.json /src/package.json
 ADD package-lock.json /src/package-lock.json
 RUN npm install
-# Install latest chrome package so we can run tests
-# Copied from https://github.com/GoogleChromeLabs/lighthousebot/blob/master/builder/Dockerfile
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
-    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
-    && apt-get update \
-    && apt-get install -y google-chrome-unstable --no-install-recommends \
-    && rm -rf /var/lib/apt/lists/* \
-    && rm -rf /src/*.deb
 ADD . /src/
 
 # add CI flag so chrome --no-sandbox flag will be used

